tests/ics/feemarket_test.go

```diff
@@ -0,0 +1,174 @@
+package ics
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	feemarkettypes "github.com/skip-mev/feemarket/x/feemarket/types"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
+
+	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/client"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	simtestutil "github.com/cosmos/cosmos-sdk/testutil/sims"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	gaiaApp "github.com/cosmos/gaia/v16/app"
+)
+
+const (
+	LargeMsgNumber = 1000
+	LargeFeeAmount = 1000000000000000
+	LargeGasLimit  = simtestutil.DefaultGenTxGas * 100
+)
+
+type FeeMarketTestSuite struct {
```

Comment:
Can you rename the folder?

Please make it `tests/integration` instead of `tests/ics` since it's no longer testing just ICS.